pages/_app.js

```diff
@@ -5,7 +5,9 @@ import { useApollo } from '../lib/apolloClient'
 
 export const theme = {
   colors: {
-    accent: '#0070f3'
+    primary: '#3C76f0',
```

Comment:
Please take a look at how theme variables are organized in `solidus_starter_fronted`. I think we should try to replicate this structure: https://github.com/nebulab/solidus_starter_frontend/blob/master/app/assets/stylesheets/spree/frontend/variables/_colors.scss